<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>soccer_window</class>
 <widget class="QWidget" name="soccer_window">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>916</width>
    <height>592</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="maximumSize">
   <size>
    <width>916</width>
    <height>592</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Play Soccer</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>Untitled1.png</normaloff>Untitled1.png</iconset>
  </property>
  <property name="layoutDirection">
   <enum>Qt::LeftToRight</enum>
  </property>
  <widget class="QPushButton" name="start_game">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>370</y>
     <width>41</width>
     <height>31</height>
    </rect>
   </property>
   <property name="maximumSize">
    <size>
     <width>171</width>
     <height>41</height>
    </size>
   </property>
   <property name="text">
    <string>Off</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="switch_fields">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>300</y>
     <width>171</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Switch ATK/DEF Area</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="battery_level_group">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>480</y>
     <width>181</width>
     <height>101</height>
    </rect>
   </property>
   <property name="title">
    <string>Battery Levels:</string>
   </property>
   <widget class="QProgressBar" name="gandalf_battery_level">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>161</width>
      <height>16</height>
     </rect>
    </property>
    <property name="value">
     <number>100</number>
    </property>
   </widget>
   <widget class="QProgressBar" name="leona_battery_level">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>161</width>
      <height>16</height>
     </rect>
    </property>
    <property name="value">
     <number>100</number>
    </property>
   </widget>
   <widget class="QProgressBar" name="presto_baterry_level">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>70</y>
      <width>161</width>
      <height>16</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="value">
     <number>100</number>
    </property>
   </widget>
   <zorder>gandalf_battery_level</zorder>
   <zorder>leona_battery_level</zorder>
   <zorder>presto_baterry_level</zorder>
   <zorder>cam_id_spinBox</zorder>
   <zorder>fps_label</zorder>
   <zorder>cam_id_label</zorder>
   <zorder>fps_lcd</zorder>
   <zorder>change_strategy</zorder>
   <zorder>start_game</zorder>
   <zorder>start_game_2</zorder>
   <zorder>switch_fields</zorder>
   <zorder>strategy_options</zorder>
   <zorder>optionsGroupBox</zorder>
   <zorder>read_parameters</zorder>
  </widget>
  <widget class="QGroupBox" name="robots_roles_group">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>480</y>
     <width>120</width>
     <height>101</height>
    </rect>
   </property>
   <property name="title">
    <string>Robots Roles:</string>
   </property>
   <widget class="QLabel" name="gandalf_role_label">
    <property name="geometry">
     <rect>
      <x>8</x>
      <y>30</y>
      <width>101</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>     -- None --</string>
    </property>
   </widget>
   <widget class="QLabel" name="leona_role_label">
    <property name="geometry">
     <rect>
      <x>8</x>
      <y>50</y>
      <width>91</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>     -- None --</string>
    </property>
   </widget>
   <widget class="QLabel" name="presto_role_label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>70</y>
      <width>101</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>    -- None --</string>
    </property>
   </widget>
  </widget>
  <widget class="QLabel" name="gandalf_label">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>500</y>
     <width>59</width>
     <height>14</height>
    </rect>
   </property>
   <property name="text">
    <string>Gandalf:</string>
   </property>
  </widget>
  <widget class="QLabel" name="leona_label">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>530</y>
     <width>59</width>
     <height>14</height>
    </rect>
   </property>
   <property name="text">
    <string>Leona:</string>
   </property>
  </widget>
  <widget class="QLabel" name="presto_label">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>560</y>
     <width>59</width>
     <height>14</height>
    </rect>
   </property>
   <property name="text">
    <string>Presto:</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="optionsGroupBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>200</y>
     <width>191</width>
     <height>141</height>
    </rect>
   </property>
   <property name="title">
    <string>Options:</string>
   </property>
   <widget class="QCheckBox" name="show_field_areas_checkbox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>101</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Show Areas</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="show_rnames_checkBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>111</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Show Names</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="show_rcentroids_checkbox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>70</y>
      <width>131</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Show Centroids</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="show_visionlogs_checkbox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>89</y>
      <width>141</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Show Vision Logs</string>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="read_parameters">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>50</y>
     <width>141</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Read Parameters</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="detection_status_group">
   <property name="geometry">
    <rect>
     <x>590</x>
     <y>480</y>
     <width>141</width>
     <height>101</height>
    </rect>
   </property>
   <property name="title">
    <string>Detection:</string>
   </property>
   <widget class="QLabel" name="gandalf_detec_label">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>30</y>
      <width>91</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Not Detected</string>
    </property>
   </widget>
   <widget class="QLabel" name="leona_detec_label">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>50</y>
      <width>91</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Not Detected</string>
    </property>
   </widget>
   <widget class="QLabel" name="presto_detec_label">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>70</y>
      <width>91</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Not Detected</string>
    </property>
   </widget>
   <widget class="QLabel" name="gandalf_detec_col_label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>21</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color : red</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="leona_detec_col_label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>21</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color : red</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="presto_detec_col_label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>70</y>
      <width>21</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color : red</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="gandalf_loss_rate">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>30</y>
      <width>31</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="leona_loss_rate">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>50</y>
      <width>31</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="presto_loss_rate">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>70</y>
      <width>31</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QLabel" name="cam_id_label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>410</y>
     <width>91</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Cam ID:</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="cam_id_spinBox">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>410</y>
     <width>47</width>
     <height>23</height>
    </rect>
   </property>
   <property name="maximum">
    <number>999</number>
   </property>
   <property name="value">
    <number>0</number>
   </property>
  </widget>
  <widget class="QPushButton" name="start_game_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>50</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Game</string>
   </property>
  </widget>
  <widget class="QGraphicsView" name="game_view">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>10</y>
     <width>671</width>
     <height>461</height>
    </rect>
   </property>
   <property name="maximumSize">
    <size>
     <width>721</width>
     <height>471</height>
    </size>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>740</x>
     <y>480</y>
     <width>161</width>
     <height>91</height>
    </rect>
   </property>
   <property name="title">
    <string>Other Info:</string>
   </property>
   <widget class="QLabel" name="ball_detec_label">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>30</y>
      <width>111</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Ball not found</string>
    </property>
   </widget>
   <widget class="QLabel" name="ball_detec_col_label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>21</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color : red</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="serial_status_col_label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>60</y>
      <width>21</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color : red</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="serial_status_label">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>60</y>
      <width>91</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Serial Closed</string>
    </property>
   </widget>
  </widget>
  <widget class="QLCDNumber" name="fps_lcd">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>440</y>
     <width>81</width>
     <height>23</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">Background-color: #000;</string>
   </property>
  </widget>
  <widget class="QLabel" name="fps_label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>440</y>
     <width>31</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>FPS:</string>
   </property>
  </widget>
  <widget class="QPushButton" name="change_strategy">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>150</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Change</string>
   </property>
  </widget>
  <widget class="QComboBox" name="strategy_options">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>150</y>
     <width>121</width>
     <height>31</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>ArrowCursor</cursorShape>
   </property>
   <property name="locale">
    <locale language="English" country="UnitedStatesVirginIslands"/>
   </property>
  </widget>
  <widget class="QLabel" name="gamelbl">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>20</y>
     <width>67</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Game:</string>
   </property>
  </widget>
  <widget class="QLabel" name="strategylbl">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>100</y>
     <width>81</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Strategy:</string>
   </property>
  </widget>
  <widget class="QLabel" name="currentStrategylbl">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>120</y>
     <width>91</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Current: </string>
   </property>
  </widget>
  <widget class="QLabel" name="cameralbl">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>370</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Set Camera: </string>
   </property>
  </widget>
  <zorder>optionsGroupBox</zorder>
  <zorder>start_game</zorder>
  <zorder>switch_fields</zorder>
  <zorder>battery_level_group</zorder>
  <zorder>robots_roles_group</zorder>
  <zorder>gandalf_label</zorder>
  <zorder>leona_label</zorder>
  <zorder>presto_label</zorder>
  <zorder>read_parameters</zorder>
  <zorder>detection_status_group</zorder>
  <zorder>cam_id_label</zorder>
  <zorder>cam_id_spinBox</zorder>
  <zorder>start_game_2</zorder>
  <zorder>game_view</zorder>
  <zorder>groupBox</zorder>
  <zorder>fps_lcd</zorder>
  <zorder>fps_label</zorder>
  <zorder>change_strategy</zorder>
  <zorder>strategy_options</zorder>
  <zorder>gamelbl</zorder>
  <zorder>strategylbl</zorder>
  <zorder>currentStrategylbl</zorder>
  <zorder>cameralbl</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
